apps/api/src/services/sendgrid_handler.py

```diff
@@ -114,3 +124,31 @@ async def send_email(
     except HTTPStatusError as e:
         log.exception("During SendGrid processing: %s", e)
         raise RuntimeError("Could not send email with SendGrid")
+
+
+async def send_decision_email(
+    sender_email: Tuple[str, str], applicant_batch: dict[tuple[str, EmailStr], Decision]
```

Comment:
Thought [non-blocking]: the format of the batch data here might be a little inconvenient to construct for the consumer. I'll think if there's a way to make it simpler, e.g. by using a `Protocol`.